source/index.js

```diff
@@ -36,6 +37,31 @@ var initWidget = function (node, options) {
     return widget;
 };
 
+var initLikely = function (nodes, options) {
+    var realNodes;
+    var realOptions;
+
+    if (Array.isArray(nodes)) {
+        // An array of nodes was passed
+        realNodes = nodes;
+        realOptions = options;
+    }
+    else if (nodes instanceof Node) {
+        // A single node was passed
+        realNodes = [nodes];
+        realOptions = options;
+    }
+    else {
+        // Options were passed, or null/undefined was passed, or the function was called without arguments
+        realNodes = dom.findAll('.' + config.name);
+        realOptions = options || nodes;
```

Comment:
While developing, I’ve somehow experienced a case when I passed `undefined` as `nodes`, and Likely didn’t recognize `options`. I think it’s irrelevant now, thanks.

As to tests, I have a couple of ideas, and I’ll try to elaborate. It’s better to test this code with unit tests instead of integration ones actually.